index.js

```diff
@@ -26,12 +25,12 @@ function explorer(loopbackApplication, options) {
     apiInfo: loopbackApplication.get('apiInfo') || {}
   });
 
-  var app = express();
+  var loopback = loopbackApplication.loopback;
```

Comment:
Well, not really, that's why I added the property :)

`require('loopback')` should work in all applications that have `loopback-explorer` sitting next to `loopback` in the same `node_modules` folder, which is probably >90% of apps out there.

We can do `var loopback = loopbackApplication.loopback || require('loopback');` to support typical users of pre-1.9 versions, but we still need to bump up the major version.

On the other hand, 1.9 was released on Jun 25, 2014, I would hope that most people have already moved on to either 1.10 or 2.0. Those who are still on pre-1.9 are preferring stability over new features, in which case they probably won't mind that explorer 1.x is no longer developed.